src/documentation/ntfs_to_netex_france_specs.md

```diff
@@ -283,3 +284,72 @@ Line/@id | lines.txt | line_id | see [id formatting](#id-of-objects)
 Line/PublicCode | lines.txt | line_code | If the code line_code is empty, this node is not created.
 Line/Name | lines.txt | line_name | 
 
+## calendriers.xml
+
+Each `service_id` produce a set of three objects:
+- `DayType`
+- `DayTypeAssignment`
+- `UicOperatingPeriod`
+
+### Top level structure
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<GeneralFrame 
+		id="FR:GeneralFrame:NETEX_CALENDRIER:<stop_provider_code>" 
+		version="any">
+	<ValidBetween>
+		<FromDate>2020-01-01T00:00:00Z</FromDate>
+		<ToDate>2020-12-30T23:59:59Z</ToDate>
+	</ValidBetween>
+	<members>
+		<!-- One DayType for each 'service_id' -->
+		<DayType />
+		<!-- One DayTypeAssignment for each 'service_id' to link
+		     a DayType and a UicOperatingPeriod -->
+		<DayTypeAssignment />
+		<!-- One UicOperatingPeriod for each 'service_id' -->
+		<UicOperatingPeriod />
+	</members>
+</GeneralFrame>
+```
+
+### ValidBetween
+
+Netex field | NTFS file | NTFS field | Note
+ValidBetween/FromDate | feed_infos.txt | feed_start_date | use ISO-8601 date-time format, using `T00:00:00Z` for the time
+ValidBetween/ToDate | feed_infos.txt | feed_end_date | use ISO-8601 date-time format, using `T23:59:59Z` for the time
+
+### DayType
+
+The `DayType` must exist even if intentionally left empty (it is referred to by
+`ServiceJourney`).
+
+Netex field | NTFS file | NTFS field | Note
+--- | --- | --- | ---
+DayType/@id | calendar.txt or calendar_dates.txt | service_id | see [id formatting](#id-of-objects)
+DayType/@version | | | fixed value `any`
+
+### DayTypeAssignement
+
+Netex field | NTFS file | NTFS field | Note
+--- | --- | --- | ---
+DayTypeAssignement/@id | calendar.txt | service_id | see [id formatting](#id-of-objects)
+DayTypeAssignement/@version | | | fixed value `any`
+DayTypeAssignement/@order | | | fixed value `0`
+DayTypeAssignement/OperatingPeriodRef/@ref | calendar.txt | service_id | see [id formatting](#id-of-objects) with `OperatingPeriod` as object type
+DayTypeAssignement/DayTypeRef/@ref | calendar.txt | service_id | see [id formatting](#id-of-objects) with `DayType` as object type
+
+### UicOperatingPeriod
+
+Netex field | NTFS file | NTFS field | Note
+--- | --- | --- | ---
+UicOperatingPeriod/@id | calendar.txt | service_id | see [id formatting](#id-of-objects) with `OperatingPeriod` as object type
+UicOperatingPeriod/@version | | | fixed value `any`
+UicOperatingPeriod/FromDate | calendar.txt | start_date | use ISO-8601 date-time format, using `T00:00:00Z` for the time
+UicOperatingPeriod/ValidDayBits | | | see (1) below
+
+**(1) Sequence for ValidDayBits**
+
+This is a string sequence of `0` (inactive days) and `1` (active days). The
+sequence should contains as many `0/1` as there is days in the period.
```

Comment:
Sorry to bother you again, but can you make explicit what the first digit stands for ? the start or the end ?

Reply:
Done in 315dd5d82a41a33dc1b95b5aec33764d2043984c.